cypress/integration/homework/calculator.spec.js

```diff
@@ -0,0 +1,158 @@
+// <reference types="Cypress" />
+
+import HomePage from './classes/HomePage.js'
+
+var homePage = new HomePage
+
+var website = "https://testsheepnz.github.io/BasicCalculator";
+
+var numberField1 = "#number1Field";
+var numberField2 = "#number2Field";
+var clearButton = "#clearButton";
+var buildDropDown = "#selectBuild";
+var actionDropDown = "#selectOperationDropdown";
+var action = "";
+
+describe('Calculator Homepage', () => {
+    describe('Addition', () => {
+        it.only("Test addition on every version of a calculator", () => {
+            action = "Add";
+            cy.visit(website)
+            cy.get(actionDropDown).select(action)
+            for (var i = 1; i <= 9; i++) {
+                context(`Build ${i}`, () => {
```

Comment:
From cypress test writing perspective **it** is the smallest test unit i.e. one separate test. That is why you should not use _context, describe_ or another _it_ inside of it.